nextflow.config

```diff
@@ -169,23 +154,35 @@ profiles {
         executor.name           = 'local'
         executor.cpus           = 4
         executor.memory         = 8.GB
+        process {
```

Comment:
I didn't think of that in our phaseimpute pipeline. But this is a really nice way of doing it !